java.exercise/src/test/java/DataProviderTests.java

```diff
@@ -0,0 +1,71 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class DataProviderTests {
+	@Test
+	public void canGetStateName() {
+		var expectedStateName = "Texas";
```

Comment:
Insert lines between the three A's (Arrange, Act, Assert) for each of your tests.